examples/tests/valid/new_in_static.test.w

```diff
@@ -27,7 +47,36 @@ let scope = new c.Construct();
 let bucket = MyClass.createBucket(scope);
 let bucket2 = createBucket();
 let my = MyClass.createMyClass(scope);
+let my2 = MyClass.createMyClassWithImplicitScope("from-root");
+let bucket3 = MyClass.createBucketWithImplicitScope();
+let my3 = MyClass.staticMehtodThatCallsAnotherStaticMethod();
 
-test "play with bucket" {
+// Call instance that instantiates stuff using static methods (should use the instance's scope)
+my.instanceMethod();
+
+// Test calling a static JSII method (no implicit scope should be passed)
+assert(jsii_fixture.JsiiClass.staticMethod("foo") == "Got foo");
+
+test "play with buckets" {
   assert(bucket.list().length == 0);
-}
\ No newline at end of file
+  assert(bucket3.list().length == 0);
+}
+
+// Call static method from a class's ctor
+class FooParent {
+  field: MyClass?;
+  new(myclass: MyClass?) {
+    this.field = myclass;
+  }
+}
+class Foo extends FooParent {
+  new() {
+    super(
+      // TODO: unsupported yet, see https://github.com/winglang/wing/issues/6016
+      // MyClass.createMyClassWithImplicitScope("from-ctor-before-super")
+      nil
+    );
+    MyClass.createMyClassWithImplicitScope("from-ctor");
```

Comment:
I guess in this case we can just pass `$scope` instead of `this`, no?

Reply:
Right, the challenge is correctly detecting this case and **not** passing `$scope` to all calls after `super()`.